plugins/action/dtc/fabric_check_sync.py

```diff
@@ -41,7 +41,7 @@ def run(self, tmp=None, task_vars=None):
         fabric = self._task.args["fabric"]
 
         ndfc_response = self._execute_module(
-            module_name="cisco.dcnm.dcnm_rest",
+            module_name="cisco.nd.nd_rest",
```

Comment:
Check `task_vars` for the `ansible_network_os` and make the module call decision based on that.